### PR TITLE
Switch to topkg

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

**Comment by hannesm** [comment]
great... you might want to revert 97a05da1a69e83d242ac69a9f428f9fba707fd4d as well to get a MirageOS-2 compatible release

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,3 @@
-Copyright (c) 2013 Citrix Inc
```
Comment (hannesm) [inline LICENSE:1]: you can remove LICENSE, since there's LICENSE.md now

File: _tags [hunk 2]
```diff
@@ -1,29 +1,6 @@
...
-<lib/*.ml{,i,y}>: pkg_io-page.unix
-<lib/*.ml{,i,y}>: pkg_ipaddr
-<lib/*.ml{,i,y}>: pkg_lwt
-<lib/*.ml{,i,y}>: pkg_lwt.unix
-<lib/*.ml{,i,y}>: pkg_mirage-types
-<lib/*.ml{,i,y}>: pkg_threads
-<lib/*.ml{,i,y}>: pkg_vmnet.lwt
-# OASIS_STOP
-true: debug, annot, bin_annot
+true: bin_annot, debug, safe_string
```
Comment (hannesm) [inline _tags:1]: debug is added automatically by topkg, no need to have it explicit here

File: opam [hunk 3]
```diff
@@ -1,19 +1,18 @@
 opam-version: "1.2"
-name: "mirage-net-macosx"
-maintainer: "Anil Madhavapeddy <anil@recoil.org>"
-authors: "Anil Madhavapeddy <anil@recoil.org>"
-homepage: "https://github.com/mirage/mirage-net-macosx"
+
+maintainer:  "Anil Madhavapeddy <anil@recoil.org>"
+authors:     "Anil Madhavapeddy <anil@recoil.org>"
```
Comment (hannesm) [inline opam:4]: this is somehow out of sync with LICENSE.md ;)

**Comment by samoht** [comment]
Subsumed by #10

**Pull request closed without merging**